### PR TITLE
octanifies ci-sequence-block-header component.

### PR DESCRIPTION
**Pull request by stopfstedt**

### CODE REVIEW

File: app/components/curriculum-inventory-sequence-block-header.js [hunk 1]
```diff
@@ -1,56 +1,36 @@
...
-        this.set('blockTitle', newBlock.get('title'));
-        this.set('sequenceBlock', newBlock);
-      } else {
-        await reject();
-      }
-    },
+  @action
+  revertTitleChanges() {
+    const block = this.args.sequenceBlock;
+    this.title = block.title;
```
Comment (jrjohnson) [inline app/components/curriculum-inventory-sequence-block-header.js:21]: `block` seems like an unnecessary local variable now that a `.get` is no longer needed.

File: tests/integration/components/curriculum-inventory-sequence-block-header-test.js [hunk 2]
```diff
@@ -1,140 +1,97 @@
...
   });
 
   test('read-only mode for block in when it can not be updated', async function(assert) {
-    const block = EmberObject.create({
-      title: 'Block title',
-      report: {
-      }
-    });
+    const block = this.server.create('curriculum-inventory-sequence-block', { title: 'Block title' });
     this.set('sequenceBlock', block);
```
Comment (jrjohnson) [inline tests/integration/components/curriculum-inventory-sequence-block-header-test.js:25]: This works for now, but `block` here isn't actually an EmberData model, it's a mirage value object. Should be `const blockModel = await this.owner.lookup('service:store').find('curriculum-inventory-sequence-block', block.id);` to be test everything as it is for a user.

File: app/components/curriculum-inventory-sequence-block-header.js (hunk 1 above)
Comment (stopfstedt) [reply app/components/curriculum-inventory-sequence-block-header.js]: good catch. handled.

File: tests/integration/components/curriculum-inventory-sequence-block-header-test.js (hunk 2 above)
Comment (stopfstedt) [reply tests/integration/components/curriculum-inventory-sequence-block-header-test.js]: handled. :+1:

**Pull request merged**